guide/14-deep-learning/working-with-multispectral-data.ipynb

```diff
@@ -119,10 +119,13 @@
     "\n",
```

Comment:
<p>Capitalize 'W' in 'we can use...' (line 1)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>